### DISCUSSION TITLE
Error occurred when executing AIO_Preprocessor:

### DISCUSSION THREAD

**Discussion by mcyyx1** [Fannovel16/comfyui_controlnet_aux#326] [Q&A]
![微信截图_20240506152100](https://github.com/Fannovel16/comfyui_controlnet_aux/assets/169030980/6110b0f2-d6ed-4555-8a5f-deb1e977e8a5)
how to solve this problem

**Comment by lornaQL** [comment]
![1](https://github.com/Fannovel16/comfyui_controlnet_aux/assets/169351409/77b8a91b-5139-4262-a8cb-d0efcb1aa967)
Encountered the same problem

**Comment by ELlenDing127** [comment]
我也是这个问题，请问解决了吗？
